mongodbatlas/online_archives.go

```diff
@@ -192,12 +192,24 @@ type OnlineArchive struct {
 	ClusterName     string                 `json:"clusterName,omitempty"`
 	CollName        string                 `json:"collName,omitempty"`
 	CollectionType  string                 `json:"collectionType,omitempty"`
-	Criteria        *OnlineArchiveCriteria `json:"criteria,omitempty"`
 	DBName          string                 `json:"dbName,omitempty"`
 	GroupID         string                 `json:"groupId,omitempty"`
-	PartitionFields []*PartitionFields     `json:"partitionFields,omitempty"`
-	Paused          *bool                  `json:"paused,omitempty"`
 	State           string                 `json:"state,omitempty"`
+	Paused          *bool                  `json:"paused,omitempty"`
+	Criteria        *OnlineArchiveCriteria `json:"criteria,omitempty"`
+	Schedule        *OnlineArchiveSchedule `json:"schedule,omitempty"`
+	PartitionFields []*PartitionFields     `json:"partitionFields,omitempty"`
+}
+
+// OnlineArchiveSchedule represents the frequency and duration when archiving process occurs.
+type OnlineArchiveSchedule struct {
+	DayOfMonth  int32  `json:"dayOfMonth,omitempty"`
+	DayOfWeek   int32  `json:"dayOfWeek,omitempty"`
+	EndHour     int32  `json:"endHour,omitempty"`
+	EndMinute   int32  `json:"endMinute,omitempty"`
+	StartHour   int32  `json:"startHour,omitempty"`
+	StartMinute int32  `json:"startMinute,omitempty"`
```

Comment:
do any of this allow zero(0)? omit empty will ignore it so they may need to be pointers

Reply:
probably EndMinute needs to allow for zero, not sure if day of the week is zero=monday or not

Reply:
I didn't check it. Thanks! I will update the fields if needed

Reply:
`DayOfMonth` and `DayOfWeek` start from 1 the other can have 0. Thanks!

Reply:
updated

Reply:
@andreaangiolillo LGTM
Reference: https://github.com/mongodb/atlas-sdk-go/blob/78b3662b91f20be6d8eed26f89944a36c4eb2ed3/admin/model_online_archive_schedule.go#L13